libbeat/scripts/Makefile

```diff
@@ -113,7 +113,7 @@ crosscompile: $(GOFILES)
 .PHONY: check
 check: check-headers python-env prepare-tests ## @build Checks project and source code if everything is according to standard
 	@go vet ${GOPACKAGES}
-	@go get $(GOIMPORTS_REPO)
+	@go get $(ES_BEATS)/vendor/$(GOIMPORTS_REPO)
```

Comment:
I think this change and the one below needs to be un-done as per https://github.com/elastic/beats/commit/db89540c67ae4a4530e70d1e991d62f8d9c868d8.